### PR TITLE
Add cards and about to our portfolio

### PR DESCRIPTION
**Pull request by tanveerisonline**
1. Added cards to our portfolio mobile version website.
2. Added about to our mobile version website.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -79,3 +78,168 @@ nav {
...
+
+.card {
+  background: #fff;
+  border-radius: 10px;
+  min-height: 3em;
+  border: 1px solid #c6c6c6;
+  padding: 15px 15px;
+  margin-bottom: 70px;
+  display: grid;
+}
```
Comment (Santiago220991) [inline styles.css]: I think you did a great job here however as part of the requirements of the project you should use flexbox to align the items inside of each card. Kindly changes this, please. :pray: :pray: 
image

File: styles.css [hunk 2]
```diff
@@ -79,3 +78,168 @@ nav {
...
+
+.about {
+  display: grid;
+  row-gap: 15px;
+  padding: 0 15px;
+}
+
+.headline_title {
+  color: #172b4d;
+}
```
Comment (Santiago220991) [inline styles.css]: Kindly ensure that your project sticks as much as possible to the Figma Template provided. Kindly make sure you are using the same font size provided by the Figma template. Change this, please. :eyes: :eyes: 
Your design
image

Figma design
image

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @tanveerisonline, 
I'm Santiago your reviewer for this PR. :wave: :wave: 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
:heavy_check_mark: Good PR summary.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Followed GitHub flow.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @tanveerisonline, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**